theano/misc/pycuda_example.py

```diff
@@ -183,13 +183,14 @@ def perform(self, node, inputs, out):
 class PycudaElemwiseSourceModuleOp(GpuOp):
     nin = property(lambda self: self.scalar_op.nin)
     nout = property(lambda self: self.scalar_op.nout)
+    __props__ = ("scalar_op", "inplace_pattern", "name")
```

Comment:
Actually, in the same file the `ThunkOp` i didn't have name and i earlier added thinking I forgot 😄